packages/grpc-js/test/test-server.ts

```diff
@@ -604,3 +621,259 @@ describe('Generic client and server', () => {
     });
   });
 });
+
+describe('Compressed requests', () => {
+  const testServiceHandlers: TestServiceHandlers = {
+    Unary(call, callback) {
+      callback(null, { count: 500000 });
+    },
+
+    ClientStream(call, callback) {
+      let timesCalled = 0;
+
+      call.on('data', () => {
+        timesCalled += 1;
+      });
+
+      call.on('end', () => {
+        callback(null, { count: timesCalled });
+      });
+    },
+
+    ServerStream(call) {
+      const { metadata, request } = call;
+
+      for (let i = 0; i < 5; i++) {
+        call.write({ count: request.message.length });
+      }
+
+      call.end();
+    },
+
+    BidiStream(call) {
+      call.on('data', (data: Request__Output) => {
+        call.write({ count: data.message.length });
+      });
+
+      call.on('end', () => {
+        call.end();
+      });
+    }
+  };
+
+  describe('Test service client and server with deflate', () => {
+    let client: TestServiceClient;
+    let server: Server;
+    let assignedPort: number;
+
+    before(done => {
+      server = new Server();
+      server.addService(
+        testServiceGrpcObject.TestService.service,
+        testServiceHandlers
+      );
+      server.bindAsync(
+        'localhost:0',
+        ServerCredentials.createInsecure(),
+        (err, port) => {
+          assert.ifError(err);
+          server.start();
+          assignedPort = port;
+          client = new testServiceGrpcObject.TestService(
+            `localhost:${assignedPort}`,
+            grpc.credentials.createInsecure(),
+            {
+              'grpc.default_compression_algorithm': 1
```

Comment:
The test code should use the `CompressionAlgorithms` enum when setting this option, both for clarity and to demonstrate that the enum works as intended.

Reply:
Good catch, thanks.